identity/app/conf/Modules.scala

```diff
@@ -1,38 +0,0 @@
-package conf
-
-import com.tzavellas.sse.guice.ScalaModule
-import com.google.inject.Provides
-import com.gu.identity.cookie.{IdentityKeys, PreProductionKeys, ProductionKeys}
-
-
-class ProdModule extends ScalaModule {
```

Comment:
this is now handled here https://github.com/guardian/frontend/pull/13571/commits/b567fb13a556e444b8b6ed1719dfb94aca2cf32d#diff-9eb1db9165a23dea3adc8ceceba582c5R12